dask/bag/core.py

```diff
@@ -4,6 +4,7 @@
 import operator
 import uuid
 import warnings
+from dask.sizeof import sizeof
```

Comment:
Nitpick: Could you please make this a relative import and move it down a few lines to be with the rest of the Dask imports 

https://github.com/dask/dask/blob/69a647ff2c80941a71afc11fadd76dbdddba48de/dask/bag/core.py#L39-L63

For reference, most modules in the codebase follow a pattern of stdlib imports, then third-party imports, then Dask imports. Not a big deal, but just wanted to point out this convention.